spidermon/contrib/scrapy/monitors/monitors.py

```diff
@@ -596,7 +596,7 @@ def _get_tags_to_filter(self):
             return {}
```

Comment:
**suggestion (non-blocking):** We might want to return empty lists for these two empty cases, though we don't really use them (as None is passed to the query instead).